awx/main/analytics/collectors.py

```diff
@@ -309,6 +309,25 @@ def events_table(since, full_path, until, **kwargs):
     return _copy_table(table='events', query=events_query, path=full_path)
 
 
+@register('events_table_playbook_stats', '1.1', format='csv', description=_('playbook on stats records'), expensive=True)
+def events_table_playbook_stats(since, full_path, until, **kwargs):
+    events_query = '''COPY (SELECT main_jobevent.id, 
```

Comment:
@jctanner isn't this data already captured in https://github.com/ansible/awx/pull/8614/files#diff-5e7230fef85166a18967e2216535cd734d66de8b26983c303981e3c82b6fa1afR284 ?

Reply:
@ryanpetrello the difference is that this gets the full event payload for playbook_on_stats events.

Reply:
I see.  Is there anything sensitive in this payload?  Is there some way we could combine both queries into one, but only conditionally return event data if the event actually *is* `playbook_on_stats`.  I ask because at large scale, this is a *very* expensive/slow query.

Reply:
It will contain all inventory hostnames. If there's some sort of conditional magic in sql to add a field when the value of another field is a certain string, the the other collector could be updated. However, that is beyond my level of ability.

Reply:
Something like this might work?

```sql
SELECT event, (CASE WHEN event = 'playbook_on_stats' THEN event_data END) FROM main_jobevent;
```

Reply:
@jctanner @ryanpetrello yeah, I think getting it in 1 query like this will be more effective on tower side (we'll do only 1 index scan of events)

Reply:
I will try to modify the other collector with that SQL example and revise the PR.

Reply:
@ryanpetrello it seems to work, ty!